newsfragments/589.changed.md

```diff
@@ -0,0 +1 @@
+Don't push chart OCI images for every PR, only tags and main.
```

Comment:
```suggestion
Don't push chart OCI images for every PR, only tags, main, and manual workflows triggers.
```

Reply:
Do you have objections to me going the other way and doing the below?

```suggestion
Don't push chart OCI images for every PR.
```